<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordWindow</class>
 <widget class="QDialog" name="RecordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add record</string>
  </property>
  <widget class="QFrame" name="accept_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>266</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="accept_btn">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>10</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept order</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="order_info_gBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Order Info</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>22</y>
      <width>311</width>
      <height>219</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="service_name_hLayout">
       <item>
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>Name of equipment:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="name_comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="type_repair_hLayout">
       <item>
        <widget class="QLabel" name="type_label">
         <property name="text">
          <string>Type of repair:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="type_repair_comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="accept_date_hLayout">
       <item>
        <widget class="QLabel" name="accept_date_label">
         <property name="text">
          <string>Acceptance date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="accept_dateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="complete_date_hLayout">
       <item>
        <widget class="QLabel" name="complete_date_label">
         <property name="text">
          <string>Completion date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="complete_dateEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="total_hLayout">
       <item>
        <widget class="QLabel" name="total_label">
         <property name="text">
          <string>Total:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="total_value_label">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="status_hLayout">
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="status_comboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="completed_order_name_hLayout">
       <item>
        <widget class="QLabel" name="completed_order_name_label">
         <property name="text">
          <string>Completed the order:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="completed_order_name_lEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
